setup.cfg

```
@@ -0,0 +1,22 @@
+[metadata]
+name = zppy-e3sm-chem-diags
+version = 1.0.0
+author = Hsiang-He Lee
+author_email = lee1061@llnl.gov
+description = Chemistry Diagnostics Package
```

Please suggest changes as you see fit.  I was following the repo as best I could. 

@golaz and @forsyth2 are both authors of zppy, so it is fine for the authors and emails to be a comma-separated list.  See:
https://github.com/E3SM-Project/zppy/blob/main/setup.py#L27-L28
but note that the format for `setup.py` and `setup.cfg` are different.

---

I wonder if we might want to add `E3SM` to the description:
```suggestion
author = Hsiang-He Lee
author_email = lee1061@llnl.gov
description = E3SM Chemistry Diagnostics Package
```